docs/using/authenticating-to-github.md

```diff
@@ -0,0 +1,35 @@
+# Authenticating to GitHub
+
+## How to login to GitHub
```

Comment:
The action is sign in, so all our docs should use the same action name and not mix login and sign in